tokens/0x8b353021189375591723e7384262f45709a3c3dc.yaml

```diff
@@ -0,0 +1,30 @@
+---
+addr: '0x8b353021189375591723e7384262f45709a3c3dc'
+decimals: 18
+description: >-
+  We're building an efficient blockchain infrastructure for decentralized applications, token issuance
+  and integration.
+links:
+- Bitcointalk: https://bitcointalk.org/index.php?topic=2021785.0
+- Blog: https://medium.com/tomocoin
+- Email: mailto:info@tomocoin.io
+- Facebook: https://www.facebook.com/tomocoin
+- Github: https://github.com/TOMOCOIN
+- Linkedin: https://www.linkedin.com/in/shekharbhusannavar/
+- Linkedin: https://www.linkedin.com/in/le-ho-cfa-55922055/
+- Linkedin: https://www.linkedin.com/in/chu-duc-minh-1773a229/
+- Linkedin: https://www.linkedin.com/in/chancejdu/
+- Linkedin: https://www.linkedin.com/in/longvuong22
+- Linkedin: https://www.linkedin.com/in/antoine-najjarin-904828b5/
+- Linkedin: https://www.linkedin.com/in/thanhson1085
+- Linkedin: https://www.linkedin.com/in/crypt-xo-androklis-polymenis-6a7137b2/
+- Linkedin: https://www.linkedin.com/in/tung-hoang-719b4657/
+- Linkedin: https://www.linkedin.com/in/minkimd14/
+- Linkedin: https://www.linkedin.com/in/limroger/
+- Linkedin: https://www.linkedin.com/in/hoang-nguyen-a493b158/
```

Comment:
Unintentional personal linkedin profiles?

Reply:
Didn't know if we were including team member LIs. Guess not. I'll remove.